shared/js/abp-preprocessed.es6.js

```diff
@@ -48,9 +48,13 @@ let lists = {
     }
 }
 
-// these are defined in trackers.js
-easylists = lists.easylists
-whitelists = lists.whitelists
+function getEasylists () {
+    return lists.easylists
+}
+
+function getWhitelists () {
+    return lists.whitelists
+}
```

Comment:
Because this file is currently one of two files that get browserified, all variables that get defined with `var`, `let` or `const` don't leak onto the global scope. (Compare that with e.g. `trackers.js` where all variable definitions at the top end up on the global scope.)

Before this change, `easylists` and `whitelists` didn't have a variable definition so that places them straight on the global scope.

`abp`, `constants` and `defaultSettings` (top of the file) also leak on the global scope, but those will be easier to swap over with a `require` statement once we add Browserify.